components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -205,4 +212,16 @@ private void removeSessionDataFromCache(String sessionDataKey) {
         OIDCSessionDataCacheKey cacheKey = new OIDCSessionDataCacheKey(sessionDataKey);
         OIDCSessionDataCache.getInstance().clearCacheEntry(cacheKey);
     }
+
+    /**
+     * Returns the OpenIDConnect User Consent.
+     *
+     * @return
+     */
+    private static boolean getOpenIDConnectSkipeUserConsent() {
+
+        boolean errorPageUrl = OAuthServerConfiguration.getInstance().getOpenIDConnectSkipeUserConsentConfig();
+        return errorPageUrl;
+
+    }
 }
```

Comment:
For the patch release it's ok to keep this method here. But for the master this method should be moved to OIDCSessionManagementUtil.